### PR TITLE
feat: change temp logo

### LINKED ISSUE #80: Update logo
**Problem**
current logo was a temporarily solution

**Solution**
Add new logo

The image files are available [here](https://drive.google.com/drive/u/0/folders/15PMR3aDCH567x2p5cqBroIVDQ7LDmWot).

### PR DESCRIPTION
**Pull request by delavaler**
Closes #80 .

- [x] I have reviewed my code in "Files changed" tab.
- [ ] I have re-read the issue and this PR fully resolves it.

### CODE REVIEW

File: app/views/layouts/ats/application.html.slim [hunk 1]
```diff
@@ -16,7 +16,6 @@ html
     link href="/icon#{'-dev' if Rails.env.development?}.svg" rel="icon" type="image/svg+xml"
     link href="/apple-touch-icon.png" rel="apple-touch-icon"
     link href="/site.webmanifest" rel="manifest"
-    link color="#d52b1e" href="/safari-pinned-tab.svg" rel="mask-icon"
```
Comment (delavaler) [inline app/views/layouts/ats/application.html.slim:19]: According to [this](https://evilmartians.com/chronicles/how-to-favicon-in-2021-six-files-that-fit-most-needs), no need for safari icon. Also, all online converters removed this icon from the generated results

File: public/android-chrome-512x512.png
Comment (ksnitsky) [inline public/android-chrome-512x512.png:1]: The new icon looks blurry (like a stretched png), is that supposed to be the case?

File: public/android-chrome-192x192.png
Comment (ksnitsky) [inline public/android-chrome-192x192.png:1]: It's blurry too

File: public/icon-dev.svg
Comment (ksnitsky) [inline public/icon-dev.svg:1]: But this one looks okay, except for that part
image

File: public/icon.svg
Comment (ksnitsky) [inline public/icon.svg:1]: Why is the icon here 92x74 rather than using the same as `app/assets/images/ats-icon.svg`?

**Review by ksnitsky — APPROVED** [review]

**Pull request merged**